docs/source/features/cloud_model_cache.md

```diff
@@ -0,0 +1,62 @@
+# Cloud Model Cache
+
+## What is Cloud Model Cache
+
+The Cloud Model Cache is a system where Olive stores intermediate models in Azure Blob Storage. Whenever a Pass is executed, Olive checks if the same input model has been processed before and if the corresponding output model is available in Azure Blob Storage. If the output model is found, Olive directly uses it for the next step. With this feature, you can easily share output models with others.
+
+This feature currently only supports native Huggingface models as input.
+
+## How to use Cloud Model Cache
+
+### Install Extra Dependencies
+
+You can install the necessary dependencies by running:
+
+```shell
+pip install olive-ai[azureml]
+```
+
+### Azure Blob Storage
+
+Olive uses Azure Blob Storage for the Cloud Model Cache, so you'll need a prepared container for this feature. Learn more about Azure Blob Storage [here](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction).
+
+Once your container is ready, locate the account URL and container name on your storage account page. The account URL typically looks like `https://<account_url>.blob.core.windows.net`.
+
+Please make sure you have logged in Azure with command:
+
+```shell
+az login
+```
+
+### Cloud Cache Configuration
+
+Add `cloud_cache_config` to Engine configuration:
+
+* `cloud_cache_config`:
+  * `enable_cloud_cache [bool]`: Whether to enable cloud cache. Default by `True`.
+  * `account_url [str]`: The account url from the Azure Storage account.
+  * `container_name [str]`: The container name to store model cache.
+  * `upload_to_cloud [bool]`: Whether to upload output model to the cloud cache for this workflow run. Default by `True`.
+
+Here is an example configuration:
+
+```json
+{
+    "engine: {
+        ...
```

Comment:
You can use `//...` instead to pass the doc build

Reply:
Also, `"engine"` is no longer required. The engine config params can be passed directly to the top level dict.